modules/Bio/EnsEMBL/Compara/PipeConfig/Metazoa/ProteinTrees_conf.pm

```diff
@@ -48,36 +48,28 @@ sub default_options {
         'division'   => 'metazoa',
 
         # homology_dnds parameters:
-        'taxlevels' => ['Drosophila' ,'Hymenoptera', 'Nematoda'],
+        'taxlevels' => ['Eukaryota'],
 
         # GOC parameters:
-        'goc_taxlevels' => ['Diptera', 'Hymenoptera', 'Nematoda'],
+        'goc_taxlevels' => ['Eukaryota'],
 
         # HighConfidenceOrthologs parameters:
         # In this structure, the "thresholds" are for resp. the GOC score, the WGA coverage and %identity
+
         'threshold_levels' => [
             {
-                'taxa'          => [ 'Euteleostomi', 'Ciona' ],
-                'thresholds'    => [ 50, 50, 25 ],
-            },
-            {
-                'taxa'          => [ 'Aculeata', 'Anophelinae', 'Caenorhabditis', 'Drosophila', 'Glossinidae', 'Onchocercidae' ],
+                'taxa'          => [ 'Penaeus', 'Thoracicalcarea', 'Dhapnia', 'Apinae', 'Haliotis', 'Echinacea', 'Asteroidea', 'Anthozoa', 'Scleractinia' ],
```

Comment:
Could the name of the third taxon from the left here include a typo? Should it be spelled `Daphnia`?